recipe/build.sh

```diff
@@ -66,8 +64,6 @@ cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
       -DBUILD_UTILITIES=ON \
       -DENABLE_DOXYGEN=OFF \
       -DENABLE_LOGGING=ON \
-      -DCMAKE_C_FLAGS_RELEASE=${CFLAGS} \
-      -DCMAKE_C_FLAGS_DEBUG=${CFLAGS} \
```

Comment:
I don't recall but I believe we had reasons to add those :-/

@mingwandroid may remember...